src/go/k8s/config/crd/bases/redpanda.vectorized.io_consoles.yaml

```diff
@@ -507,6 +507,11 @@ spec:
                 properties:
                   enabled:
                     type: boolean
+                  useNodeCA:
+                    description: Indication on whether to use the node CA as trust
+                      when connecting to the schema registry. If not set, the public
+                      CAs will be used.
+                    type: boolean
```

Comment:
A Node is a kubernetes resource. If you don't mean a Kubernetes node, please use a different word.

Reply:
See https://github.com/redpanda-data/redpanda/pull/11211#discussion_r1219744668